crest/Assets/Crest/Crest/Scripts/LodData/RegisterLodDataInput.cs

```diff
@@ -148,7 +148,17 @@ public virtual void Draw(LodDataMgr lodData, CommandBuffer buf, float weight, in
                     buf.SetGlobalVector(sp_DisplacementAtInputPosition, Vector3.zero);
                 }
 
-                buf.DrawRenderer(_renderer, _material);
+                for (var i = 0; i < _renderer.sharedMaterials.Length; i++)
+                {
```

Comment:
Out of interest, why doesn't DrawRenderer do all this automatically?

Reply:
I have no idea. No one has questioned it from what I can see. I did a test to see if it is the same for mesh renderers and it is.

Reply:
I'm thinking of changing null/continue to an assertion as empty slots is a problem. How does that sound?

Reply:
Is it a problem? If so then sounds fine to me

Reply:
Yeah. Done. Thanks